portfolio/src/main/webapp/index.html

```diff
@@ -37,13 +37,16 @@ <h5 id="blurb">(that's pronounced "key-fur"!)</h5>
       </div>
       <div class="column right">
         <h3 class="header">About Me</h3>
-        <p>Lorem ipsum blah blah put info hereeeeeeee</p>
+        <p>My name is Keiffer Acoba, and I'm a <a href="https://www.cs.cmu.edu/">Computer Science</a> student studying at Carnegie Mellon University, with minors in Machine Learning and Business Adminstration. I grew up in Long Island, New York, but am proud of my Filipino heritage where my parents were born.</p>
+        <p>I found that I've always been interested in computation and how exactly it uses mathematics and clever engineering to work. Having grown up in a community where STEM was not heavily emphasized, I enjoy teaching and showing others just how fascinating tech can be. Currently I serve as Director of Operations of CMU's ScottyLabs, one of the largest tech-based organizations on campus that specializes in hosting events such as our big annual hackathon, TartanHacks. With my position I can continue to help spread the excitement and positive potential that technology can bring.</p>
+        <p>Beyond technical and academic interests, my personal hobbies also include playing video games (I pretty much love anything from Nintendo), music composition, and, more recently, pixel art. Having these creative outlets help me relax a little from STEM.</p>
+        <p>Feel free to look at my portfolio and résumé on this website! You can also check out some places you can find me in social media below.</p>
         <ul class="links">
-          <li class="link"><a href="#" target="_blank"><ion-icon name="mail-outline"></ion-icon></ion-icon></a></li>
-          <li class="link"><a href="#" target="_blank"><ion-icon name="logo-facebook"></ion-icon></ion-icon></ion-icon></a></li>
-          <li class="link"><a href="#" target="_blank"><ion-icon name="logo-instagram"></ion-icon></ion-icon></ion-icon></a></li>
-          <li class="link"><a href="#" target="_blank"><ion-icon name="logo-linkedin"></ion-icon></ion-icon></ion-icon></a></li>
-          <li class="link"><a href="#" target="_blank"><ion-icon name="logo-octocat"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="mailto:keiffer.acoba@gmail.com" target="_blank"><ion-icon name="mail-outline"></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://www.facebook.com/keiffer.acoba" target="_blank"><ion-icon name="logo-facebook"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://instagram.com/keifferacoba" target="_blank"><ion-icon name="logo-instagram"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://www.linkedin.com/in/keiffer-acoba-90426416b" target="_blank"><ion-icon name="logo-linkedin"></ion-icon></ion-icon></ion-icon></a></li>
+          <li class="link"><a href="https://github.com/keiffer01" target="_blank"><ion-icon name="logo-octocat"></ion-icon></ion-icon></ion-icon></a></li>
```

Comment:
Are those extra "</ion-icon>" necessary? Could you please verify?

Wondering if <ion-icon name="mail-outline" /> works in this case if nothing added in betwee.

Reply:
Which quotes are you referring to?

Reply:
aha I forgot I am writing HTML. it doesn't show "<\/ion-icon>"

Reply:
According to the usage documentation, it looks like you need the opening and closing tags separate: https://ionicons.com/usage/